runner/src/main.rs

```diff
@@ -79,17 +92,21 @@ async fn main() -> Result<()> {
     runs.add(1, &[KeyValue::new("command", args.command.name())]);
 
     info!(?args.command, ?args.otlp_endpoint, "starting runner");
-    let success = match args.command {
+    let success = match args.command.clone() {
         Command::Bootstrap(opts) => bootstrap(opts).await?,
         Command::Simulate(opts) => simulate(opts).await?,
+        Command::GenerateLoad(opts) => simulate_load(opts).await?,
         Command::Noop => CommandResult::Success,
     };
-
-    // Flush traces and metrics before shutdown
-    shutdown_tracer_provider();
-    metrics_controller.force_flush()?;
-    drop(metrics_controller);
-    shutdown_meter_provider();
+    if !matches!(args.command, Command::GenerateLoad(_)) {
+        // Flush traces and metrics before shutdown
+        shutdown_tracer_provider();
+        if let Some(metrics_controller) = metrics_controller.clone() {
+            metrics_controller.force_flush()?;
+        }
+        drop(metrics_controller);
+        shutdown_meter_provider();
```

Comment:
the drop/shutdown should maybe be in the `if let` statement since it's not used otherwise but it doesn't look like it's caused an issue

Reply:
I think we should add back the `drop(metrics_controller);`. Probably doesn't matter now but it might cause some confusion when metrics are fixed and prevent them from showing up as expected.